code/modules/mob/living/carbon/species.dm

```diff
@@ -410,6 +410,38 @@
 
 	H.stop_tail_wagging(1)
 
+/datum/species/vulpkanin
+	name = "Vulpkanin"
+	icobase = 'icons/mob/human_races/r_vulpkanin.dmi'
+	deform = 'icons/mob/human_races/r_vulpkanin.dmi'
+	path = /mob/living/carbon/human/vulpkanin
+	default_language = "Galactic Common"
+	language = "Canilunzt"
+	primitive = /mob/living/carbon/monkey/vulpkanin
+	tail = "vulptail"
+	unarmed_type = /datum/unarmed_attack/claws
+	darksight = 8
+	slowdown = -1
+	brute_mod = 1.1
+	burn_mod = 1.1
```

Comment:
Claws don't do extra damage compared to punches; that's one thing that got standardized away a while back.

In any event, this is definitely something I'll be keeping a close eye on, especially considering some of the things you mentioned (which will make it harder to gather data on since hardly anyone here uses meth here, routinely).

But yeah, it's a concern of mine, as well.